Gemfile.lock

```diff
@@ -108,6 +108,7 @@ GEM
     parallel (1.21.0)
     parser (3.0.3.2)
       ast (~> 2.4.1)
+    pg (1.3.0.rc1)
```

Comment:
this is probably caused by `>= 1.2.3` - we're installing release candidate: there gonna be dragons!

Reply:
My bad entirely, didn't check it before pushing; thanks for being this vigilant!